shared-bindings/i2cperipheral/I2CPeripheral.c

```diff
@@ -85,10 +85,7 @@ STATIC mp_obj_t i2cperipheral_i2c_peripheral_make_new(const mp_obj_type_t *type,
     uint8_t *addresses = NULL;
     unsigned int i = 0;
     while ((item = mp_iternext(iterable)) != MP_OBJ_STOP_ITERATION) {
-        mp_int_t value;
-        if (!mp_obj_get_int_maybe(item, &value)) {
-            mp_raise_TypeError_varg(translate("can't convert %q to %q"), MP_QSTR_address, MP_QSTR_int);
-        }
+        mp_int_t value = mp_arg_validate_type_int(item, MP_QSTR_address);
         if (value < 0x00 || value > 0x7f) {
             mp_raise_ValueError(translate("address out of bounds"));
```

Comment:
`address` is not the name of the arg but we can use that qstr here anyway.
```suggestion
        mp_arg_validate_int_range(value, 0x00, 0x7f, MP_QSTR_address);
```

Reply:
done